docs/install.md

```diff
@@ -0,0 +1,228 @@
+
+#### CLI
+
+You can download Chef InSpec via script:
+```
+curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P inspec
+```
+
+#### Habitat
+
+Use Chef Habitat to install Chef InSpec, or include Chef InSpec in your own Chef Habitat packages. Chef InSpec is available [on the Habitat Builder Depot](https://bldr.habitat.sh/#/pkgs/chef/inspec).
+
+Use the following command to install the Chef InSpec package:
+
+```
+$ hab pkg uninstall chef/inspec
+```
+
+See about available flag options in the [hab pkg install documentation](https://www.habitat.sh/docs/habitat-cli/#hab-pkg-install).
+
+### Windows
+
+#### Installer
+
+Start by downloading the latest [Chef InSpec package](https://downloads.chef.io/inspec) relevant to your operating system version.
+
+Once downloaded, double-click the `.msi` file to launch the installer and follow the prompts.
+
+#### Powershell
+
+Use the following command to install Chef InSpec via Powershell:
+
+```powershell
+. { iwr -useb https://omnitruck.chef.io/install.ps1 } | iex; install -project inspec
+```
+
+Once Chef InSpec is installed, run `inspec version` to verify that the installation was successful.
+
+#### Habitat
+
+Use Chef Habitat to install Chef InSpec, or include Chef InSpec in your own Chef Habitat packages. 
+You can find [Chef InSpec on the Depot here](https://bldr.habitat.sh/#/pkgs/chef/inspec).
+
+### Linux
+
+#### CLI
+
+For Ubuntu, use the following command to install Chef InSpec:
+
+```
+sudo dpkg -i /path/to/inspec.deb
+```
+
+For Red Hat Enterprise Linux, use the following command to install Chef InSpec:
+
+```
+sudo rpm -U /path-to/inspec.rpm
+```
+
+For SUSE Linux Enterprise Server, use the following command to install Chef InSpec: 
+
+```
+zypper install /path-to/inspec.rpm
+```
+
+#### Habitat
+
+Use Chef Habitat to install Chef InSpec, or include Chef InSpec in your own Chef Habitat packages. 
+You can find [Chef InSpec on the Depot here](https://bldr.habitat.sh/#/pkgs/chef/inspec).
+
+
+## Update Chef InSpec
+
+### macOS
+
+#### Homebrew
+
+Run the following command to update Chef InSpec on Homebrew:
+
+```
+brew cask upgrade inspec
+```
+
+#### CLI
+
+#### Habitat
+
+
+
+### Windows
+
+#### Installer
+
+#### Powershell
+
+#### Habitat
+
+
+
+### Linux
+
+#### CLI
+
+To update Chef InSpec in Ubuntu, use the following command to update Chef InSpec:
+
+```
+sudo dpkg -i /path/to/inspec.deb
+```
+
+For Red Hat Enterprise Linux, use the following command to update Chef InSpec:
+```
+sudo rpm -U /path-to/inspec.rpm
+```
+
+For SUSE Linux Enterprise Server, use the following command to update Chef InSpec: 
+
+```
+
+```
+
+#### Habitat
+
+
+## Uninstall Chef InSpec
+
+Chef InSpec can be uninstalled using the steps below that are appropriate for the method of Chef InSpec installation.
+
+### macOS
+
+#### Homebrew
+
+Use the following *destructive* command to remove the Chef InSpec standalone Homebrew package:
+
+```
+$ brew uninstall --force chef/chef/inspec
```

Comment:
Chances are, if you needed to `sudo` under the covers to install, you'll prolly need to do so to uninstall?

Reply:
I'm somewhat confused by this comment. The content above is adapted from the homebrew documentation.

Reply:
Also, I had to triple check and this command did not work. I updated the PR accordingly to something that does work! :)